src/main/java/com/kubetetris/Main.java

```diff
@@ -21,31 +28,31 @@
 
     private static KubernetesAccessor k8S;
 
-    public static void main(String[] args) throws InterruptedException {
+    public static void main(String[] args) throws Exception {
 
         CliArgs cliArgs = new CliArgs(args);
 
         //Create SystemController and nodes and pods
         SystemController systemController = new SystemControllerImpl();
 
         //Fetch Nodes from K8S
-        k8S = new KubernetesAccessorImpl();
+        k8S = new KubernetesAccessorImplV2();
 
         //If the command line arguments contains the -podsAlreadyCreated switch anywhere, the switchPresent() method will return true.
         //If not, the switchPresent() method will return false.
         boolean podsAlreadyCreated = cliArgs.switchPresent("-podsAlreadyCreated");
-        List<Node> inputNodes = fetchNodes(podsAlreadyCreated);
+        List<Node> inputNodes = fetchNodes(true);
 
         //Update the system snapshot with latest nodes update with pods
         inputNodes.forEach(node -> systemController.addNode(node));
 
-        placeMyWorkload(systemController);
+        //placeMyWorkload(systemController);
 
         System.out.println("***********************************************************************************");
         System.out.println("Trying to balance the cpu/memory consumption across nodes");
         System.out.println("***********************************************************************************");
 
-        Thread.sleep(5000);
+        //Thread.sleep(5000);
```

Comment:
Uncomment

Reply:
I am making further changes to either run balancer or workload placement, but not both together.